data_steward/resource_files/fields/_deactivated_participants.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
I would suggest adding this definition to a sub-directory in `fields`.  Possibly something like `static_tables`, `lookup_tables`, or even `pipeline_tables`.

Reply:
Created a subdirectory called `lookup_tables` and put `_deactivated_participants` in it.